remote-method/index.js

```diff
@@ -0,0 +1,315 @@
+        type: 'confirm',
+        default: true
+      },
+      {
+        name: 'description',
+        message: 'Description for method:'
+      }
+    ];
+    this.prompt(prompts, function(answers) {
+      this.methodName = answers.methodName;
+      this.isStatic = answers.isStatic;
+      this.description = answers.description;
+      done();
+    }.bind(this));
+  },
+
+  delimEndpoints: function() {
+    this.log('Let\'s add some HTTP Endpoints now.\n');
+  },
+
+  askForEndpoints: function() {
+    var done = this.async();
+    this.log('Enter an empty path when done.');
+
+    var prompts = [
+      {
+        name: 'httpPath',
+        message: 'Enter the path of this endpoint:',
+        required:true
+      }
+    ];
+    this.prompt(prompts, function(answers) {
+      if (answers.httpPath == null || answers.httpPath === '') {
+        return done();
+      }
+      var subprompts = [
+        {
+          name: 'httpVerb',
+          message: 'HTTP verb:',
+          type: 'list',
+          choices: ['post', 'put', 'delete', 'get', {
+            name: '(other)',
+            value: null
+          }]
+        },
+        {
+          name: 'customHttpVerb',
+          message:
+          'Enter the custom http verb',
+          validate: validateName,
+          when: function(answers) {
+            return answers.httpVerb === null;
+          }
+        }
+      ];
+
+      var httpPath = answers.httpPath;
+
+      this.prompt(subprompts, function(answers) {
+        https.push({
+          'httpPath': httpPath,
+          'httpVerb': answers.httpVerb || answers.customHttpVerb
+        });
+        this.log('\nLet\'s add another endpoint.');
+        this.askForEndpoints();
+      }.bind(this));
+    }.bind(this));
+  },
+
+  delimAccepts: function() {
+    this.log('Let\'s add some Accept Arguments now.\n');
+  },
+
+  askForAccepts: function() {
+    var done = this.async();
+    this.log('Enter an empty name when done.');
+
+    var prompts = [
+      {
+        name: 'acceptsArg',
+        message: 'Enter name for accept argument:',
+        required: true,
+        validate: validateName
+      }
+    ];
+    this.prompt(prompts, function(answers) {
+      if (answers.acceptsArg == null || answers.acceptsArg === '') {
+        return done();
+      }
+      var subprompts = [
+        {
+          name: 'acceptsType',
+          message: 'The type of accept argument:',
+          type: 'list',
+          choices: typeChoices
+        },
+        {
+          name: 'acceptsRequired',
+          message: 'Required?',
+          type: 'confirm',
+          default: false
+        },
+        {
+          name: 'acceptsDes',
+          message: 'Description of accept argument:',
+          required: true
+        },
+        {
+          name: 'httpSource',
+          message: 'Mapping to http source:',
+          default: 'body',
+          type: 'list',
+          choices: [{
+            name: '(default) No mapping',
```

Comment:
This is not correct. When there is no http mapping specified, strong-remoting will search all places for the argument, see the [source](https://github.com/strongloop/strong-remoting/blob/32efdc433f8bd12f1ecc7872c855216b99c0911e/lib/http-context.js#L246-L258).

That's why I'm proposing to use an option name `(default)`.